api/config.go

```diff
@@ -0,0 +1,98 @@
+/*
+event-reporter - report events to the Cacophony Project API.
```

Comment:
This should be updated to match the actual project (yes we need to extract the API client code!)